src/lib/tool_providers.ml

```diff
@@ -494,7 +506,7 @@ let default_toolkit
     star_tool ~host ~meta_playground;
     stringtie_tool ~host ~meta_playground;
     cufflinks_tools ~host ~meta_playground;
-    hisat_tool ~host ~meta_playground;
+    hisat_tool ~host ~meta_playground ~version:`V_0_1_6_beta;
```

Comment:
I think it's fine to have other versions in the default toolkit. What seems more confusing is saying that the default tool does not exist in the default toolkit.